Do not run type check (mypy) because it requires projects we do not have
packaged yet:

pygobject-stubs
shtab
types-pywin32

--- keyring-25.4.1/tox.ini.orig
+++ keyring-25.4.1/tox.ini
@@ -1,5 +1,5 @@
 [testenv]
-description = perform primary checks (tests, style, types, coverage)
+description = perform primary checks (tests, style, coverage)
 deps =
 setenv =
 	PYTHONWARNDEFAULTENCODING = 1
@@ -11,7 +11,6 @@
 	check
 	cover
 	enabler
-	type
 
 [testenv:diffcov]
 description = run tests and check that diff from main is covered
